smartcontractkit/external-adapters-js
# Use separate makeConfig for dxfeed-secondary

## Closes [DF-18751](https://smartcontract-it.atlassian.net/browse/DF-18751)

## Description
Add `makeConfig()` to `makeExecute()` in `dxfeed-secondary` adapter to ensure it _does not_ use symbol overrides from `dxfeed`. This works because the overrides object keys off of the lowercase adapter name, and without this change, the config name for `dxfeed-secondary` returns `DXFEED`.



## Steps to Test
- `yarn && yarn setup && yarn test packages/sources/dxfeed-secondary/test/integration`
- `curl` against any symbol shown [here](https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/sources/dxfeed/src/config/symbols.json) to see that `dxfeed-secondary` no longer uses the overrides present there for price responses

## Quality Assurance



[DF-18751]: https://smartcontract-it.atlassian.net/browse/DF-18751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
